proposed/http-message.md

```diff
@@ -1236,6 +1236,16 @@ interface UriInterface
      * This method MUST return a string; if no path is present it MUST return
      * the string "/".
      *
+     * The value returned MUST be returned in percent-encoded form, but MUST NOT
+     * double-encode any characters. To determine what characters to encode,
+     * please refer to RFC 3986, Sections 2 and 3.3.
+     *
+     * If the value should include a slash ("/") not intended as delimiter
```

Comment:
I would try to make clear that this is just an example. The same applies for the other reserved characters as well.